www/front_src/src/PollerWizard/pollerStep2/index.tsx

```diff
@@ -1,55 +1,119 @@
 import * as React from 'react';
 
-import { connect } from 'react-redux';
+import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router';
-
-import { useRequest, postData } from '@centreon/ui';
-
-import Form from '../forms/poller/PollerFormStepTwo';
+import { useUpdateAtom, useAtomValue } from 'jotai/utils';
+import { pick } from 'ramda';
+
+import {
+  Typography,
+  Button,
+  FormControlLabel,
+  Checkbox,
+  CircularProgress,
+} from '@mui/material';
+
+import {
+  postData,
+  useRequest,
+  MultiAutocompleteField,
+  SelectField,
+  SelectEntry,
+} from '@centreon/ui';
+
+import { pollerAtom, setWizardDerivedAtom, PollerData } from '../PollerAtoms';
+import { useStyles } from '../../styles/partials/form/PollerWizardStyle';
 import routeMap from '../../reactRoutes/routeMap';
-import { setPollerWizard } from '../../redux/actions/pollerWizardActions';
-import { WizardFormProps } from '../models';
+import {
+  labelAdvancedServerConfiguration,
+  labelLinkedRemoteMaster,
+  labelLinkedadditionalRemote,
+  labelOpenBrokerFlow,
+  labelPrevious,
+  labelApply,
+} from '../translatedLabels';
+import { Props, PollerOrRemoteList } from '../models';
 
 const getPollersEndpoint =
   './api/internal.php?object=centreon_configuration_remote&action=getRemotesList';
 const wizardFormEndpoint =
   './api/internal.php?object=centreon_configuration_remote&action=linkCentreonRemoteServer';
-interface Props
-  extends Pick<WizardFormProps, 'goToNextStep' | 'goToPreviousStep'> {
-  pollerData: Record<string, unknown>;
-  setWizard: (data) => void;
-}
 
+interface stepTwoFormData {
```

Comment:
interfaces should start with an Uppercase
```suggestion
interface StepTwoFormData {
```